modal/runner/containers/vllm_unified.py

```diff
@@ -139,19 +142,33 @@ def __init__(self):
     gpu=modal.gpu.A100(count=1, memory=40),
     concurrent_inputs=32,
 )
+
+# A re-mapping of model names to their respective quantized models.
```

Comment:
any way of avoiding code duplication / making the keys in this dict constants that are shared with the other definition?

Reply:
i had it that way originally but our short line length config made it look funky

will change it back for safety